### DISCUSSION TITLE
Initial Installation

### DISCUSSION THREAD

**Discussion by Elder-HVAC-Man** [dahlb/ha_carrier#45] [General]
I have installed the integration through HACS and need help finding where to add the information to get it started.  Nothing shows up anywhere and the instructions only state available through the UI.

Thanks

**Comment by dahlb** [comment]
After you install it with hacs, then you have to add it through the normal integration interface

**Comment by Elder-HVAC-Man** [comment]
Thanks.  Uninstalled it, rebooted and re-installed it again.  Rebooted and I am not seeing anything in the main integrations screen or in the add-ons.  It does show up in the HACS integration screen, but nothing appears to accept any input.  Can this be setup via the yaml in the config file?

Larry Novak(281) 850-4686 

    On Friday, August 11, 2023 at 11:08:10 AM CDT, Brendan Dahl ***@***.***> wrote:  
 
 


After you install it with hacs, then you have to add it through the normal integration interface

**Reply by dahlb** [reply]
![Screenshot_20230811-124531.png](https://github.com/dahlb/ha_carrier/assets/54117/e3ed852a-8607-4a75-a712-7426373826f9)

![Screenshot_20230811-124540.png](https://github.com/dahlb/ha_carrier/assets/54117/2168a01c-d492-4222-bbfb-f856b9a71630)

You should be able to use the add integration button and then type 'carrier' to find it, then it will ask for your carrier login. No it is currently not configured through yaml only the UI wizards.

**Comment by Elder-HVAC-Man** [comment]
Thank you very much for your help.  Found it and installed!

Larry
 

    On Friday, August 11, 2023 at 11:48:01 AM CDT, Brendan Dahl ***@***.***> wrote:  
 
 






You should be able to use the add integration button and then type 'carrier' to find it, then it will ask for your carrier login. No it is currently not configured through yaml only the UI wizards.
